packages/flutter_tools/test/commands.shard/permeable/build_aar_test.dart

```diff
@@ -190,9 +194,18 @@ void main() {
 
   group('Gradle', () {
     Directory tempDir;
+    AndroidSdk mockAndroidSdk;
+    String gradlew;
+    FakeProcessManager processManager;
+    String flutterRoot;
 
     setUp(() {
       tempDir = globals.fs.systemTempDirectory.createTempSync('flutter_tools_packages_test.');
+      mockAndroidSdk = FakeAndroidSdk(globals.fs.directory('irrelevant'));
```

Comment:
Overriding `directory` isn't necessary.
```suggestion
      mockAndroidSdk = FakeAndroidSdk();
```

Reply:
done